docs/packages/pkg/EnumGroup.rst

```diff
@@ -0,0 +1,20 @@
+.. spelling::
+
+    EnumGroup
+
+.. index:: unsorted ; EnumGroup
+
+.. _pkg.EnumGroup:
+
+EnumGroup
+==========
```

Comment:
Please make it 8 for esthetic reason.

Reply:
It doesn't work with 8. It said the title underline is too short. It works with 9.

Reply:
> It doesn't work with 8

Hah, yeah, I failed to count to 9 :) My bad, sorry.

> It said the title underline is too short. It works with 9

Great, thank.